packages/gallery/tests/styleParams/overlayAnimation.spec.js

```diff
@@ -62,4 +62,26 @@ describe('options - overlayAnimation', () => {
     expect(items.length).to.be.greaterThan(0);
     driver.detach.proGallery();
   });
+  it('should have overlay animation class - "SLIDE DOWN"', async () => {
+    initialProps.options = mergeNestedObjects(initialProps.options, {
+      galleryLayout: GALLERY_CONSTS.layout.GRID,
+      overlayAnimation: GALLERY_CONSTS.overlayAnimations.SLIDE_DOWN,
```

Comment:
@yuvalab `SLIDE_LEFT` and `SLIDE_DOWN` is the only difference between these two tests. Please reuse the duplicated code.